src/app/shared/mission.model.ts

```diff
@@ -0,0 +1,12 @@
+export class Mission {
+
+  client: string;
+  title: string;
+
+  constructor(pClient: string, pTitle: string) {
```

Comment:
prefer `missionClient` and `missionTitle`, I don't understand what is the `p`, maybe for `parameters` ?

Reply:
Exactly